controller/base.php

```diff
@@ -69,5 +70,17 @@ public function __construct(helper $helper, ideas $ideas, linkhelper $link_helpe
 		$this->php_ext = $php_ext;
 
 		$this->user->add_lang_ext('phpbb/ideas', 'common');
+
+		// Don't let using Ideas if it's not been properly configured
```

Comment:
yeah but the constructor should not really perform any operations, it should only set up the class environment. It may be a little duplication but it makes it clear in each controller that they are dependent on something when reading the code.